src/flac/main.c

```diff
@@ -1278,98 +1278,109 @@ void show_help(void)
 	printf("      --output-prefix=STRING   Prepend STRING to output names\n");
-	printf("      --delete-input-file      Deletes after a successful encode/decode\n");
-	printf("      --preserve-modtime       Output files keep timestamp of input (default)\n");
+	printf("      --preserve-modtime       (default) Output files keep timestamp of input\n");
 	printf("      --keep-foreign-metadata  Save/restore WAVE or AIFF non-audio chunks\n");
-	printf("      --keep-foreign-metadata-if-present  Save/restore WAVE or AIFF non-audio\n");
-	printf("                        but not return an error when no such chunks are found\n");
+	printf("      --keep-foreign-metadata-if-present     Save/restore WAVE or AIFF non-audio\n");
+	printf("                           but not return an error when no such chunks are found\n");
 	printf("      --skip={#|mm:ss.ss}      Skip the given initial samples for each input\n");
-	printf("      --until={#|[+|-]mm:ss.ss}  Stop at the given sample for each input file\n");
-#if FLAC__HAS_OGG
-	printf("      --ogg                    Use Ogg as transport layer\n");
-	printf("      --serial-number          Serial number to use for the FLAC stream\n");
-#endif
-	printf("analysis options:\n");
-	printf("      --residual-text          Include residual signal in text output\n");
-	printf("      --residual-gnuplot       Generate gnuplot files of residual distribution\n");
-	printf("decoding options:\n");
+	printf("      --until={#|[+|-]mm:ss.ss}     Stop at the given sample for each input file\n");
+	printf("      --no-utf8-convert        Do not convert tags from local charset to UTF-8\n");
+	printf("  -s, --silent                 Do not write runtime encode/decode statistics\n");
+	printf("      --totally-silent         Do not print anything, including errors\n");
+	printf("  -w, --warnings-as-errors     Treat all warnings as errors\n");
+	printf("\n");
+	printf("Decoding options:\n");
 	printf("  -F, --decode-through-errors  Continue decoding through stream errors\n");
 	printf("      --cue=[#.#][-[#.#]]      Set the beginning and ending cuepoints to decode\n");
-	printf("encoding options:\n");
-	printf("  -V, --verify                 Verify a correct encoding\n");
-	printf("      --lax                    Allow encoder to generate non-Subset files\n");
-	printf("      --ignore-chunk-sizes     Ignore data chunk sizes in WAVE/AIFF files\n");
-	printf("      --replay-gain            Calculate ReplayGain & store in FLAC tags\n");
-	printf("      --cuesheet=FILENAME      Import cuesheet and store in CUESHEET block\n");
-	printf("      --picture=SPECIFICATION  Import picture and store in PICTURE block\n");
-	printf("  -T, --tag=FIELD=VALUE        Add a FLAC tag; may appear multiple times\n");
-	printf("      --tag-from-file=FIELD=FILENAME   Like --tag but gets value from file\n");
-	printf("  -S, --seekpoint={#|X|#x|#s}  Add seek point(s)\n");
-	printf("  -P, --padding=#              Write a PADDING block of length #\n");
+    printf("\n");
```

Comment:
A few lines here have the wrong kind of indentation I think, probably tabs instead of spaces.